packages/contracts-bedrock/src/L1/L1StandardBridge.sol

```diff
@@ -70,22 +70,35 @@ contract L1StandardBridge is StandardBridge, ISemver {
     );
 
     /// @notice Semantic version.
-    /// @custom:semver 2.0.0
-    string public constant version = "2.0.0";
+    /// @custom:semver 2.1.0
+    string public constant version = "2.1.0";
 
     /// @notice Address of the SuperchainConfig contract.
     SuperchainConfig public superchainConfig;
 
     /// @notice Constructs the L1StandardBridge contract.
-    /// @param _messenger Address of the L1CrossDomainMessenger.
-    constructor(address payable _messenger) StandardBridge(_messenger, payable(Predeploys.L2_STANDARD_BRIDGE)) {
-        initialize({ _superchainConfig: SuperchainConfig(address(0)) });
+    constructor() StandardBridge() {
+        initialize({
+            _messenger: CrossDomainMessenger(address(0)),
+            _otherBridge: StandardBridge(payable(Predeploys.L2_STANDARD_BRIDGE)),
+            _superchainConfig: SuperchainConfig(address(0))
+        });
     }
 
-    /// @notice Initializes the contract.
-    /// @param _superchainConfig Address of the SuperchainConfig contract on this network.
-    function initialize(SuperchainConfig _superchainConfig) public initializer {
+    /// @notice Initializer.
+    /// @param _messenger        Contract for the CrossDomainMessenger on this network.
+    /// @param _otherBridge      Contract for the StandardBridge on the other network.
+    /// @param _superchainConfig Contract for the SuperchainConfig on this network.
+    function initialize(
+        CrossDomainMessenger _messenger,
+        StandardBridge _otherBridge,
```

Comment:
We always know what the address of the `_otherBridge` is so we can reduce footguns by hardcoding it and removing it from the interface of `initialize`. It is the predeploys constant